mapiproxy/libmapistore/backends/indexing_mysql.c

```diff
@@ -103,21 +103,13 @@ static memcached_st *_memcached_setup(struct indexing_context *ictx,
 		return ictx->cache;
 	}
 
-	/* Initialize memcached connection */
-	if (conn_str) {
-		memc = memcached(conn_str, strlen(conn_str));
-		if (!memc) return NULL;
-	} else {
-		memc = (memcached_st *) memcached_create(NULL);
-		if (!memc) return NULL;
-
-		servers = memcached_server_list_append(servers, "127.0.0.1", 11211, &rc);
-		rc = memcached_server_push(memc, servers);
-		if (rc != MEMCACHED_SUCCESS) {
-			OC_DEBUG(0, "[ERR]: Unable to add server to memcached list\n");
-			return NULL;
-		}
+	/* Request a memcached connection */
+	memc = oc_memcached_new_connection(conn_str);
+	if (!memc) {
+		OC_DEBUG(1, "Error trying to get memcached connection");
```

Comment:
This debug message is not relevant any longer. It is superseded by the granularity of debug logs in oc_memcached_new_connection()